packages/lib/domainManager/deploymentServices/vercel.ts

```diff
@@ -28,13 +28,28 @@ export const createDomain = async (domain: string) => {
     method: "POST",
   });
 
-  const data = vercelDomainApiResponseSchema.parse(await response.json());
+  const responseJson = await response.json();
 
-  if (!data.error) {
+  const parsedResponse = vercelDomainApiResponseSchema.safeParse(responseJson);
+
+  if (!parsedResponse.success) {
```

Comment:
Fixes the crash as well as logs the faulty response which we can use to improve the schema